packages/runtime-node/src/node-environment.ts

```diff
@@ -138,3 +125,11 @@ export function createFeatureLoaders(
     }
     return featureLoaders;
 }
+
+function toRecord(options: [string, string | boolean][] | undefined) {
```

Comment:
very close to a Object.fromEntries clone, can it be removed?

Reply:
removed